exonum-java-binding-core/rust/ejb-app/TUTORIAL.md

```diff
@@ -96,3 +96,22 @@ $ cargo run -- finalize testnet/sec.toml testnet/node.toml \
 ```$sh
 $ cargo run -- run -d testnet/db -c testnet/node.toml --public-api-address 127.0.0.1:3000
 ```
+
+There are few specific optional parameters here:
+- `--jvm-args-prepend` and `--jvm-args-append`: Additional parameters for JVM that prepend and
+ append the rest of arguments. Must not have a leading dash. For example, `Xmx2G`.
+- `--jvm-debug`: Allows JVM being remotely debugged over the `JDWP` protocol. Takes a socket address as a parameter in form
+ of `HOSTNAME:PORT`. For example, `localhost:8000`.
+
+```$sh
+$ cargo run -- run -d testnet/db -c testnet/node.toml --public-api-address 127.0.0.1:3000 \
```

Comment:
I'd remove this example because it is not clear what it demonstrates (passing prepend/append or debugging the JVM)?

I think it would be better to put it in a separate section `#### Debugging the JVM` and keep only the required option:
```
#### Debugging the JVM

To enable remote debugging of Java code on a running Exonum node, 
pass `--jvm-debug` option with a socket address to connect to
from a debugger:

$ cargo run -- run -d testnet/db -c testnet/node.toml --public-api-address 127.0.0.1:3000 \
    --jvm-debug localhost:8000


Now you can debug the service using any JDWP client, such as command line
JDB or a debugger built in your IDE:

jdb -attach localhost:8000 -sourcepath /path/to/source
```

Reply:
Sounds reasonable. I was in doubt with the previous version - on one hand, I had to document new arguments for the `Run` stage, and on the other hand, they are optional thus not required to just start the service. Your variant looks more clear to me.